src/targets/gpu/device/include/migraphx/gpu/device/reduce.hpp

```diff
@@ -242,6 +242,23 @@ constexpr index_int compute_block_size(index_int n, index_int max_block_size)
     return block_size;
 }
 
+inline std::vector<index_int> get_reduce_lens(const std::vector<size_t>& input_lens,
```

Comment:
This file was refactored from previous changes, but it does not directly affect layernorm at the moment.